r_scripts/analysis/oregon_part_a.Rmd

```diff
@@ -0,0 +1,1083 @@
+---
+title: "Oregon GWR smoke association with Cardiopulmonary Emergency Room visits (PartA)"
+author: "Jingyang Liu"
+date: "April 9, 2018"
+output: html_document
+---
+
+
+
+## Overview
+
+This project researches on for the association between GWR smoke PM~2.5~ and cardiopulmonary (cardiovascular (CVD) and respiratory health outcomes) diseases for Oregon State in wildfire season (May to September) in 2013. 
+This markdown document focuses on the zip code-level analysis. 
+The analysis includes total analysis, age strata and sex strata.
+
+
+
+## Methods Description
+
+In these research, we mainly examine GWR smoke/PM~2.5~ estimations and associations with health outcomes using a time-stratified case-crossover study design. Health outcomes for a patient with a primary diagnosis of cardiopulmonary health outcomes and their date of admission  (index time) were identified. We then created counterfactual observations for each patient for the same day of the week for the entire wildfire season (MAy 1st to Sep 30th, 2013). We further limited our analyses to claims that were coded as emergency or urgent visits to eliminate bias from patients going in for elective/planned procedures. However, we found the filtered population is much larger than the population we chose in Washington, so we choose only the emegency room (hospital) visits and urgent care visits. For patients who have a index time closer to May 1, 2013 or closer to September 30, 2013, their referent observations before or after these dates will be excluded as I will not be able to assign estimates of PM~2.5~ to these referent observations. However, this is not a big issue for a time-stratified design as other referent values throughout the time period average out.  
+
+The **cardiopulmonary health outcomes** are all respiratory, asthma, COPD, cardiovascular disease, heart failure, and ischemic heart disease.
+
+As for **exposure methods**, we use Geo-Weighted Regression and subtracted off the 'Background' estimates of smoke, which I believe are the monthly averages of PM~2.5~ for a given grid. However, the 'Background' is the mean of total period from May 1st to Sep 30th 2013 so I have to assign the mean for every day. Then use the 10 units of the smoke for analysis.
+
+The **analytic method** is the conditional logistic regression model using the *survival* package in R. This regression is controlling the confounders. Each conditional logistic regression model accounts for the subject, and adjusts for temperature from the WRF-Chem model. The conditional logistic regression model \beta represents the change in risk of an event associated with a short-term 10 units' increase in exposure, and can be calculated as an average difference between exposure at the index time and a weighted average of exposure at all times.

```

Comment:
clogit controls for time-invariant subject-specific confounding like age and sex (factors that will not change over a short period of time, but could confound the association).